docs/developers-clear-containers-install.md

```diff
@@ -39,7 +39,8 @@ $ mkdir -p $GOPATH
 ```bash
 $ go get github.com/clearcontainers/runtime
 $ go get github.com/clearcontainers/proxy
```

Comment:
In fact, the runtime and proxy should now be `go get -d ...` since we need to call `make` for both.